src/ex1/style_ex1.css

```diff
@@ -0,0 +1,173 @@
+body {
+  font-family: "Dosis", sans-serif;
+  background-color: rgb(153, 203, 205);
+}
+
+.white-box {
+  position: relative;
+  margin: 0 auto;
+  width: 400px;
+  height: 600px;
+  background-color: white;
+  border-radius: 1%;
+}
+
+.main-header {
+  margin-left: 25px;
+  padding-top: 15px;
+}
+
+.task-input-wrapper {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
```

Comment:
As the relative positioning goes from left to right and top to bottom, it's a better practice to control the element's position against the existing dom on the top & left, thus using margin-left/margin-top instead of positioning the next set of elements (right-bottom).